website/docs/r/cloudfront_distribution.html.markdown

```diff
@@ -216,9 +216,11 @@ resource "aws_cloudfront_distribution" "s3_distribution" {
   # AWS Managed Caching Policy (CachingDisabled)
   default_cache_behavior {
     # Using the CachingDisabled managed policy ID:
-    cache_policy_id  = "4135ea2d-6df8-44a3-9df3-4b5a84be39ad"
-    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
-    target_origin_id = local.s3_origin_id
+    cache_policy_id        = "4135ea2d-6df8-44a3-9df3-4b5a84be39ad"
+    allowed_methods        = ["GET", "HEAD", "OPTIONS"]
+    cached_methods         = ["GET", "HEAD"]
```

Comment:
Thanks for the doc update, it will likely save someone some time in future 👍 

Referring back to my original bug report https://github.com/hashicorp/terraform-provider-aws/issues/34095, it doesn't make sense to me to have to provide a `cached_methods` attribute when caching is disabled - are GET and HEAD cached or not? I still don't know 😁 .

But looking at the Cloudfront console, I think this is a constraint coming from AWS, i.e. even when selected Cache policy = CachingDisabled, we still see above "GET and HEAD methods are cached by default.

<img width="1272" height="533" alt="image" src="https://github.com/user-attachments/assets/7e9da612-82ef-4bb0-9948-26ca51444bbe" />